dotnet-interactive.v3.ncrunchsolution

```diff
@@ -0,0 +1,6 @@
+﻿<SolutionConfiguration>
+  <Settings>
+    <AllowParallelTestExecution>False</AllowParallelTestExecution>
```

Comment:
Have you run into any issues with parallel execution that we're not handling directly in the affected tests? Typically it's better to leave this on true.

Reply:
Not at all, I disabled this when trying to figure out why this was broken. Will enable and push